cmd/mayactl/app/command/volume_info.go

```diff
@@ -23,6 +23,13 @@ var (
         `
 )
 
+//values keeps info of the values of a current address in replicaIPStatus map
+type values struct {
+	i      int
```

Comment:
i is needed here ? we already assigning below in range of `addressIPStrings`

Reply:
yes i is needed here because it is used during formation of replica info